gymnasium/dev_wrappers/rescale_action.py

```diff
@@ -0,0 +1,80 @@
+"""Wrapper for rescaling actions to within a max and min action."""
+from typing import Union
+
+import jumpy as jp
+import numpy as np
+
+import gymnasium as gym
+from gymnasium import spaces
+from gymnasium.core import ActType
+
+
+class RescaleActionV0(gym.ActionWrapper):
+    """Affinely rescales the continuous action space of the environment to the range [min_action, max_action].
+
+    The base environment :attr:`env` must have an action space of type :class:`spaces.Box`. If :attr:`min_action`
+    or :attr:`max_action` are numpy arrays, the shape must match the shape of the environment's action space.
+
+    Example:
+        >>> import gymnasium as gym
+        >>> import numpy as np
+        >>> env = gym.make('BipedalWalker-v3', disable_env_checker=True)
+        >>> env.action_space
+        Box(-1.0, 1.0, (4,), float32)
+        >>> min_action = -0.5
+        >>> max_action = np.array([0.0, 0.5, 1.0, 0.75])
+        >>> env = RescaleActionV0(env, min_action=min_action, max_action=max_action)
+        >>> env.action_space
+        Box(-0.5, [0.   0.5  1.   0.75], (4,), float32)
+        >>> RescaleAction(env, min_action, max_action).action_space == gym.spaces.Box(min_action, max_action)
+        True
+    """
+
+    def __init__(
+        self,
+        env: gym.Env,
+        min_action: Union[float, int, np.ndarray],
+        max_action: Union[float, int, np.ndarray],
+    ):
+        """Initializes the :class:`RescaleAction` wrapper.
+
+        Args:
+            env (Env): The environment to apply the wrapper
+            min_action (float, int or np.ndarray): The min values for each action. This may be a numpy array or a scalar.
+            max_action (float, int or np.ndarray): The max values for each action. This may be a numpy array or a scalar.
+        """
+        assert isinstance(
+            env.action_space, spaces.Box
+        ), f"expected Box action space, got {type(env.action_space)}"
+        assert np.less_equal(min_action, max_action).all(), (min_action, max_action)
+
+        super().__init__(env)
+        self.min_action = (
```

Comment:
In my understanding, we don't need to modify the action space as the space specifies the valid input for inputs not outputs.

Reply:
Same question as here: https://github.com/Farama-Foundation/Gymnasium/pull/150#discussion_r1032955271

Reply:
Yes no issue

Reply:
Updated both for `rescale_action` and for `clip_action` here https://github.com/Farama-Foundation/Gymnasium/pull/150/commits/f616199bd40309e2d5eac1f276975bce04537c49